File: Android/app/src/main/java/com/example/groceryapplication/StoreProfile.kt
package com.example.groceryapplication

data class StoreProfile(
    val id: String,
    val docType: String = "StoreProfile",
    val storeId: String,
    val name: String,
    val contact: Contact,
    val location: Location,
    val manager: String? = null,
    val openingHours: String? = null
) {
    data class Contact(
        val email: String,
        val phone: String
    )
    
    data class Location(
        val address1: String,
        val address2: String? = null,
        val locality: String,
        val region: String,
        val postalCode: String,
        val country: String,
        val coordinates: Coordinates? = null
    )
    
    data class Coordinates(
        val lat: Double,
        val lon: Double
    )
}

File: iOS/GroceryApp/StoreProfile.swift
import Foundation

struct StoreProfile: Identifiable, Codable {
    let id: String
    let docType: String
    let storeId: String
    let name: String
    let contact: Contact
    let location: Location
    let manager: String?
    let openingHours: String?
    
    struct Contact: Codable {
        let email: String
        let phone: String
    }
    
    struct Location: Codable {
        let address1: String
        let address2: String?
        let locality: String
        let region: String
        let postalCode: String
        let country: String
        let coordinates: Coordinates?
    }
    
    struct Coordinates: Codable {
        let lat: Double
        let lon: Double
    }
}

